fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -95,6 +95,31 @@ def run
 
       always_trace!
 
+      command "lint" do |c|
```

Comment:
it initially was 'validate-fastlane' - which i didnt get to work with symbol. -> fixed